torch/csrc/jit/codegen/cuda/lower_unroll.cpp

```diff
@@ -202,12 +199,12 @@ void UnrollPass::handle(kir::ForLoop* fl) {
   handle(inlined_loop);
   look_for_unroll_ = true;
   if (!non_trivial_pred_found_) {
```

Comment:
Do we still need this? IIUC, `non_trivial_pred_found_` is always true, isn't it?

Reply:
Ah, yes, thanks.